mitiq/rem/tests/test_post_select.py

```diff
@@ -50,7 +51,13 @@ def test_post_select_empty_measurement_result(inverted):
 
 def test_post_select_hamming_weight_less_than():
     res = MeasurementResult(
-        [[0, 1, 1], [1, 1, 0], [0, 0, 0], [0, 0, 1], [1, 1, 1]]
+        [
```

Comment:
This is another array that has been exploded thought interestingly the internal arrays were not.